### PR TITLE
Update vba.g4

### PR DESCRIPTION
**Pull request by Xenios91**
Should resolve issue #3334

### CODE REVIEW

File: vba/vba.g4 [hunk 1]
```diff
@@ -539,19 +539,17 @@ valueStmt :
 	| valueStmt WS? POW WS? valueStmt 						# vsPow
 	| MINUS WS? valueStmt 									# vsNegation
 	| PLUS WS? valueStmt 									# vsPlus
-	| valueStmt WS? DIV WS? valueStmt 						# vsDiv
-	| valueStmt WS? MULT WS? valueStmt 						# vsMult
+	| valueStmt (WS? DIV WS? | WS? MULT WS? ) valueStmt 	# vsDivMult
```
Comment (msagca) [inline vba/vba.g4:542]: You can leave `WS?` outside the parentheses:
```antlr
| valueStmt WS? (DIV | MULT) WS? valueStmt
```

**Comment by msagca** [comment]
@Xenios91 

The order of operations still isn't correct, e.g. `NOT` is below all the other logical operators while it has the highest precedence. Also, the relational operators should be grouped as well. Please refer to the table I posted in #3334.

File: vba/vba.g4 [hunk 2]
```diff
@@ -539,19 +539,17 @@ valueStmt :
...
 	| valueStmt WS? MOD WS? valueStmt 						# vsMod
-	| valueStmt WS? PLUS WS? valueStmt 						# vsAdd
-	| valueStmt WS? MINUS WS? valueStmt 					# vsMinus
-	| valueStmt WS? AMPERSAND WS? valueStmt 					# vsAmp
+	| valueStmt (WS? PLUS WS? | WS? MINUS WS?) valueStmt 	# vsAddMinus
```
Comment (KvanTTT) [inline vba/vba.g4:544]:
```suggestion
	| valueStmt WS? (PLUS | MINUS) WS? valueStmt 	# vsAddMinus
```

**Review by KvanTTT — CHANGES_REQUESTED** [review]
Thanks for the contribution but please fix issues mentioned by @msagca and me.

**Comment by Xenios91** [comment]
Alright, I believe that should resolve the issues?

**Comment by Xenios91** [comment]
Alright, I believe that should resolve the issues requested to be fixed?

**Comment by msagca** [comment]
@Xenios91 

I just noticed that the order still isn't correct. Relational operators should come just before the logical operators and below mul/div/add/sub/mod.

**Comment by Xenios91** [comment]
> @Xenios91
> 
> I've just noticed that the order still isn't correct. Relational operators should come just before the logical operators (after pow, mul, div, mod, add, sub and bitwise and).

Hows that look now?

**Comment by msagca** [comment]
@Xenios91 

Looks good to me.

**Comment by Xenios91** [comment]
awesome, thanks for the guidance.

**Review by KvanTTT — APPROVED** [review]

**Comment by KvanTTT** [comment]
Yes, everything is ok.

@Xenios91 welcome to ANTLR grammars repository!

**Pull request merged**